Source/MainWindow.cpp

```diff
@@ -41,7 +41,13 @@ BMenuBar*
 MainWindow::_PrepareMenuBar(void)
 {
 	BMenuBar* menubar = new BMenuBar("menu");
-	BMenu* menu = new BMenu(B_TRANSLATE("Edit"));
+	BMenu* menu = new BMenu(B_TRANSLATE("Weather"));

```

Comment:
I think using B_TRANSLATE_SYSTEM_NAME("Weather") here would be correct and saves a duplicated 'effort' of translation.

Reply:
Indeed, thank you. Updated with your other suggestions.